doc/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ set(DOXYGEN_ALIASES
   )
 
 set(DOXYGEN_PREDEFINED
-  "OPENVDB_VERSION_NAME=v8_2"
+  "OPENVDB_VERSION_NAME=v8_0"
```

Comment:
oops, what happened here

Reply:
Looks like I bumped only bumped the major version:

https://github.com/AcademySoftwareFoundation/openvdb/pull/898/files#diff-652f011e4f45934b3684d4596670e3847750ec3289b5092d4f8847b1ca6a75edR112

There are warnings that Doxygen spits out about the links being broken, but unfortunately they don't get turned into errors so they were silently ignored. I looked this up and there's a FAIL_ON_WARNINGS flag we can add to Doxygen, but it's only recently been introduced and I think is released or is going to be released as 1.9.0 (https://github.com/doxygen/doxygen/issues/8023). We could attempt to upgrade or maybe error through parsing the log, but I didn't go further than this - I think getting the release process automated is probably more important.